### ISSUE TITLE
Error when Repository unavailable

### ISSUE THREAD

**Issue by sawe1900** [vMarkusK/Advanced-PRTG-Sensors#95]
First of all, great script!! Thanks a lot!

When one of my Backup Repositories is unavailable, PRTG reports with:
XML: Structural error in xml file, 4 open items. -- JSON: The returned JSON does not match the expected structure (Invalid JSON.). (code: PE231)

Also see data result logfile attached. 
Is there any way to catch this error?

[Results of sensor 17494.txt](https://github.com/vMarkusK/Advanced-PRTG-Sensors/files/10210063/Results.of.sensor.17494.txt)

**Comment by vMarkusK** [comment]
`Exception calling "SyncSpaceInfoToDb" with "2" argument(s): "[REPO-02] Failed to connect to Ins
taller service." `

The script tries to update the usage of each repo on runtime. This call failed for one of your repos.

**Comment by sawe1900** [comment]
> `Exception calling "SyncSpaceInfoToDb" with "2" argument(s): "[REPO-02] Failed to connect to Ins taller service." `
> 
> The script tries to update the usage of each repo on runtime. This call failed for one of your repos.

Yes, correct. Is it possible to catch this error and display it correctly in the PRTG sensor instead of the JSON error?

**Comment by vMarkusK** [comment]
I will have a look at it. The "SyncSpaceInfoToDb" call is pretty often involved in problems, so it might be a great enhancement.
